### DISCUSSION TITLE
Remove a person not well categorized

### DISCUSSION THREAD

**Discussion by ewok2** [matiasdelellis/facerecognition#741] [Q&A] [answered]
Hello
When a face in not associated to the good person is it possible to : 
- remove it (what will happend with the picture?)
- associate it with a person already known

**Comment by Edgeberg** [comment, marked as answer]
Yes, this is done from the Files app, where you can access the add/remove functionality from the People tab in the sidebar. It would be more convenient if it could be done from within the Face Recognition app, but it can still be done.

To find the right photo in the Files app to adjust the associated person match data, you can go into the "Review Face Groups" function in the list of photos for a person, then select a photo and you will be taken to the Files app in a new browser window/tab with that photo selected.

In the People tab of the sidebar, you should see actions to the right of each face; a "this person is not <name>" action to remove the name from a face currently associated with a name, and an "Add name" action for faces without names.

**Comment by ewok2** [comment]
Thanks it works !!!
Only thing to be carefull, not "add name" in file app on people not already "named" because when you tag one, all "recognized" will be tag also :-)

Thanks you!!!

**Reply by ewok2** [reply]
Two other question (not fully linked... but not so far)
- When you select "this person is not " is there a way to avoid the pop up "explaining it wil ...."
- When I tag in File App manualy (or when Face Recognation tag) where is it written ? in the exif of the picture? if not is there a way to write it on the picture to be sure that all the manually corrections will not be loose in case of reworking of the app?

**Reply by Edgeberg** [reply]
1. I don't know
2. I don't know but I expect it would be written to the Nextcloud database, not to the EXIF in the picture file. Upgrades / reworking of the face recognition app should not endanger the metadata you've worked on - that would be disastrous!

**Comment by ewok2** [comment]
Thanks for repy
